<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BtConnector</class>
 <widget class="QDialog" name="BtConnector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listDevicesNearby">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>256</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bluetooth devices nearby :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonScan">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonConnect">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Connnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDisconnect">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRemoteDeviceInfo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_localDevice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Local device info:</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_address">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>BtConnector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
